tests/service-catalog/Dockerfile

```diff
@@ -15,11 +15,13 @@ RUN CGO_ENABLED=0 go test -tags=acceptance -c ./test -o /servicecatalog.test
 FROM alpine:3.10
 RUN apk add --no-cache curl
 
+WORKDIR /app
+
 COPY --from=builder env-tester.bin /go/bin/env-tester.bin
 COPY --from=builder ./servicecatalog.test .
```

Comment:
do you know if it matters that `servicecatalog.test` moved inside of the container from `/servicecatalog.test` to `/app/servicecatalog.test`?

Reply:
no since `.` means current work directory, which is set to `/app`.

Reply:
I know, just wondering if some app expects it to be hardcoded path. Feel free to unhold at your convenience.